include/oneapi/dnnl/dnnl.hpp

```diff
@@ -14180,8 +14180,6 @@ inline void primitive::execute(const stream &astream,
 
 /// @endcond
 
-#undef DNNL_DEFINE_BITMASK_OPS
```

Comment:
Undefining the macro here will prevent us from using the macro in dnnl_graph.hpp, when both dnnl.hpp and dnnl_graph.hpp are included. See an example here: https://github.com/uxlfoundation/oneDNN/blob/ceb0d6c8c5685ee9af1eb6c07a3508248e886673/examples/graph/sdpa.cpp#L26-L27